src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        String logo = "         z$$$$P\n" +
+                "        d$$$$\"\n" +
+                "      .$$$$$\"\n" +
+                "     z$$$$$\"\n" +
+                "    z$$$$P\n" +
+                "   d$$$$$$$$$$\"\n" +
+                "  *******$$$\"\n" +
+                "       .$$$\"\n" +
+                "      .$$\"\n" +
+                "     4$P\"\n" +
+                "    z$\"\n" +
+                "   zP\n" +
+                "  z\"\n" +
+                " /    M A V I S'\n" +
+                "^";
+        //Store added tasks to an array
+        Task[] listOfTasks = new Task[100];
+
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Mavis!");
+        System.out.println("What can I do for you?\n");
+
+        //String to store the input entered by the user
+        String inputToEcho;
+
+        //This variable stores the current number of tasks added
+        // and also uses it to place a new task in the next available spot.
+        int listOfTasksSize = 0;
+
+        Scanner in = new Scanner(System.in);
+
+        while(true) {
+            inputToEcho = in.nextLine();
+
+            if (inputToEcho.equals("bye")) {
+                break;
+            }
+
+            else if (inputToEcho.equals("list")) {
```

Comment:
else if should not be so far from the previous if block

Reply:
Will make all if-else blocks follow the K&R style, with the else trailing immediately after the end of the if statement, thank you for catching that!